src/frontend/apps/impress/src/features/auth/api/useAuthQuery.tsx

```diff
@@ -16,6 +21,16 @@ import { User } from './types';
  */
 export const getMe = async (): Promise<User> => {
   const response = await fetchAPI(`users/me/`);
+
+  if (!response.ok && response.status == 401 && canAttemptSilentLogin()) {
+    const currentLocation = window.location.href;
+    attemptSilentLogin(3600);
```

Comment:
It should be in const in the conf I think, for everybody to know that it is something that we could configure, wdyt?